searchcore/src/vespa/searchcore/proton/flushengine/flushengine.h

```diff
@@ -23,31 +23,33 @@ class FlushEngine final : public FastOS_Runnable
 public:
     class FlushMeta {
     public:
-        FlushMeta(const vespalib::string & name, uint32_t id);
+        FlushMeta(const vespalib::string& handler_name, const vespalib::string& target_name, uint32_t id);
         ~FlushMeta();
         const vespalib::string & getName() const { return _name; }
+        const vespalib::string& handler_name() const { return _handler_name; }
         vespalib::system_time getStart() const { return vespalib::system_clock::now() - std::chrono::duration_cast<vespalib::system_time::duration>(elapsed()); }
```

Comment:
rename getStart -> getStartInEpoch and make repurpose getStart to return steady_time.